src/sinks/prometheus/exporter.rs

```diff
@@ -333,8 +338,35 @@ impl MetricNormalize for PrometheusExporterMetricNormalizer {
     }
 }
 
+fn check_auth(req: &Request<Body>, auth: &Option<Auth>) -> bool {
```

Comment:
```suggestion
fn authorized(req: &Request<Body>, auth: &Option<Auth>) -> bool {
```

I think this make the meaning of the return value a bit clearer.

Reply:
Fixed